### PR TITLE
post: ng 5.1

### PR DESCRIPTION
**Pull request by cexbrayat**
WIP

- [x] change changelog URl once it's released
- [x] AoT unit tests
- [x] registerLocaleData
- [x] error messages improved
- [x] service worker enabled

### CODE REVIEW

File: _posts/2017-12-18-what-is-new-angular-5.1.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+## Compiler
+
+It's worth noting that behind the scenes, some work has been done to enable AoT unit testing.
+Currently units test are run using the JiT compiler.
+But as you may know, the Angular team is working to make this JiT compiler obsolete.
+It's been recommended for a long time to use the AoT mode in production,
+and, starting with Angular 5.0, it's no longer necessary to use JiT even in development
+as AoT has become faster (even it's still slower than JiT right now).
+The last place where JiT is required is for unit testing.
+That should no longer be the case soon, as some key pieces are falling in place in the framework.
```
Comment (jnizet) [inline _posts/2017-12-18-what-is-new-angular-5.1.md]: It would be interesting to explain how to do that with the CLI.

Comment (cexbrayat) [reply _posts/2017-12-18-what-is-new-angular-5.1.md]: It's not (yet) available :)

File: _posts/2017-12-18-what-is-new-angular-5.1.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    registerLocaleData(localeFr, 'fr-ZZ');
+
+and the french locale data will be available for the (fake) locale id `fr-ZZ`.
+
+## Service worker
+
+The `@angular/service-worker` package evolves a little,
+with the possibility to register the `ServiceWorkerModule` without crashing the application
+even if the Service Worker API is not supported by the browser.
+The `register` method also now has a new option to enable the service worker or not.
```
Comment (clacote) [inline _posts/2017-12-18-what-is-new-angular-5.1.md]: I would say "The `register` method has now also a new..."

File: _posts/2017-12-18-what-is-new-angular-5.1.md (hunk 1 above)
Comment (clacote) [inline _posts/2017-12-18-what-is-new-angular-5.1.md]: falling in place -> falling into place, I think

File: _posts/2017-12-18-what-is-new-angular-5.1.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+      }
+    }
+
+## Compiler
+
+It's worth noting that behind the scenes, some work has been done to enable AoT unit testing.
+Currently units test are run using the JiT compiler.
+But as you may know, the Angular team is working to make this JiT compiler obsolete.
+It's been recommended for a long time to use the AoT mode in production,
+and, starting with Angular 5.0, it's no longer necessary to use JiT even in development
```
Comment (clacote) [inline _posts/2017-12-18-what-is-new-angular-5.1.md]: `Angular&nbsp;5.0`

**Comment by cexbrayat** [comment]
Angular 5.1 has been released: not much new features so it's a short post.
Scheduled to tomorrow (Thursday). Would be great to have your thumbs up @jnizet @clacote !
Ebook release incoming

**Pull request merged**